### PR TITLE
feat(ui5-time-picker): improve keyboard handling support

### PR DESCRIPTION
**Pull request by tsanislavgatev**
Fixed one keyboard handling bug and added new features:
- PageUp - Hours + 1
- PageDown - Hours - 1
- PageUp + Shift - Minutes + 1
- PageDown + Shift - Minutes - 1
- PageUp + Shift + Ctrl - Seconds + 1
- PageDown + Shift + Ctrl - Seconds - 1

Related to: https://github.com/SAP/ui5-webcomponents/issues/1534

### CODE REVIEW

File: packages/main/test/specs/TimePicker.spec.js [hunk 1]
```diff
@@ -123,4 +123,52 @@ describe("TimePicker general interaction", () => {
...
+		timepicker.click();
+		timepicker.keys(['Shift', 'Control', 'PageUp']);
+		timepicker.keys('Control');
+
+		// assert
+		assert.strictEqual(timepicker.shadow$("ui5-input").getProperty("value"), "12:00:02", "The value of seconds is +1");
+		// act
+		timepicker.click();
+		timepicker.keys(['Shift', 'Control', 'PageDown']);
+		timepicker.keys('Shift');
```
Comment (ilhan007) [inline packages/main/test/specs/TimePicker.spec.js:168]: Isn't `timepicker.keys(['Shift', 'Control', 'PageDown']);` sufficient or it does not work without these:

```js
timepicker.keys('Shift');
timepicker.keys('Control');
```
*The question goes also for lines 140 and 161

Comment (tsanislavgatev) [reply packages/main/test/specs/TimePicker.spec.js:168]: As per the documentation on wdio side : https://webdriver.io/docs/api/browser/keys.html

The modifier keys should be pressed and then released(by simulating second press) to terminate their action.

File: packages/main/src/TimePicker.js [hunk 2]
```diff
@@ -708,15 +708,18 @@ class TimePicker extends UI5Element {
...
 			date.setMinutes(date.getMinutes() + incrementStep);
-		} else if (seconds) {
+		} else if (seconds && this.shouldBuildSecondsSlider) {
 			date.setSeconds(date.getSeconds() + incrementStep);
+		} else {
+			return;
 		}
 
 		this.setValue(this.formatValue(date));
+		this.fireEvent("change", { value: this.value, valid: true });
```
Comment (ilhan007) [inline packages/main/src/TimePicker.js:722]: Shouldn't we fire "change" after the user presses the "submit" button?

Comment (tsanislavgatev) [reply packages/main/src/TimePicker.js:722]: This keyboard handling functionality can be used with closed picker also.

**Pull request merged**